### PR TITLE
Create reader projections issue 11

### PR DESCRIPTION
**Pull request by Dmytry-Dyachenko**
Add reader’s projections:

```
- BookDetailsView
- DialogWindow
- HomePage
```
Replace `EmailAddress`, and `BookCoverUrl` with `spine/net/email_address.proto`, and `spine/net/url.proto` respectively.

Edit license agreement.

### CODE REVIEW

File: model/src/main/proto/javaclasses/exlibris/q/projections.proto [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+// 2. Borrowed.
+// 3. Reserved.
+//
+// (or maybe some more sorting: for example for new "hot" books).
+//
+message HomePage {
+
+    repeated Book book = 1;
+    repeated Category category = 2;
+
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/projections.proto]: Remove whitespace

**Review by alexanderkarpetc — CHANGES_REQUESTED** [review]
Delete whitespaces in values.proto

**Review by alexanderkarpetc — APPROVED** [review]
LGTM

**Review by PaulAgeyev — APPROVED** [review]
LGTM

**Pull request merged**